README.md

```diff
@@ -4,11 +4,26 @@ This library helps to manage and configure singleuser JupyterHub servers deploye
 
 # Configuration
 
-```
-profiles:
+``` profiles:
   - name: globals
     env:
       THIS_IS_GLOBAL: "This will appear in all singleuser pods"
+    # Set OpenShift node tolerations for a notebook pod. See https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/taints_tolerations.html#admin-guide-taints for more information.
```

Comment:
I would suggest adding another profile into the example with name such as "Special nodes" and moving the node_tolerations and node_affinity there - since a big chunk of YAML and may not be the first thing people are interested in (IMO that would be resources and envs) it kind of sticks out here at the top. What do you think?

Reply:
Done.